docs/gvisor.md

```diff
@@ -0,0 +1,69 @@
+## gVisor Addon
```

Comment:
nit: I'd kinda prefer this file was a README.md in the deploy/addons/gvisor directory so that we can just link to that directory and the docs will show there along with the code.

Reply:
for sure, done